src/main/java/vazkii/psi/common/spell/trick/block/PieceTrickBreakBlock.java

```diff
@@ -114,15 +114,9 @@ public static BreakEvent createBreakEvent(IBlockState state, EntityPlayer player
 	}
 
 	// Based on ForgeHooks::canHarvestBlock
-	public static boolean canHarvestBlock(Block block, EntityPlayer player, World world, BlockPos pos, ItemStack tool) {
+	public static boolean canHarvestBlock(Block block, EntityPlayer player, World world, BlockPos pos) {
```

Comment:
This is a break in binary compatibility, as it's a public static method. 

This also means that CAD tools can't mine above the CAD level, which is unintended behavior. For example, a pickaxe spell is meant to be able to mine obsidian, whereas (with a default config) a CAD spell cannot.